content/blog/2025-06-17-tmil-18/index.md

```diff
@@ -0,0 +1,107 @@
+- [xilem#992][]: Add support for multiple (top-level) desktop windows in Xilem, by Martin Fischer.
+- [xilem#1071][]: Split Xilem into modules.
+- [xilem#1079][], [xilem#1085][]: Provide the app's state to each `View` during more phases. This will allow many future architecture explorations.
+- [xilem#1078][]: Allow accessing the channel of new requests for a `worker` directly.
+- [xilem#1094][]: Refactor `lens` into its own view so that the state doesn't need to passed when constructing the view.
+- [xilem#1097][]: Small example to teach variable length lists, by Nils Martel.
+- [xilem#1102][]: Add a helper for flex rows, to improve readability of layout code.
+
+In June, we started a new initiative for Xilem, a Mastodon client example.
+This is being developed alongside Xilem to focus its development.
+
+- [xilem#1087][]: Display a single scrollable timeline.
+- [xilem#1089][], [xilem#1116][]: Add simple HTML processing to make posts more readable.
+- [xilem#1091][]: Load and display the avatar of post authors.
+- [xilem#1092][]: Improve the styling of posts.
+- [xilem#1108][]: Correctly display boosted posts.
+- [xilem#1114][]: Allow viewing replies to a post.
+
+<!-- TODO: Image of Placehero -->
+
+## Parley
+
+Parley is a text layout library.
+It handles text layout, mostly at the level of line breaking and resolving glyph positions.
+
+In June, we released Parley [0.5.0](https://github.com/linebender/parley/releases/tag/v0.5.0).
+This includes features we talked about in previous months, including restoring `Layout: Sync`, and improvements to line height handling.
+
+- [parley#376][]: An initial version of `AttributedText`.
+- [parley#381][]: Select "hard lines" rather than soft lines when triple clicking.
+- [parley#382][]: Use ui-events-winit in the Vello Editor example.
+
+## Android View
+
+[Andrdoid View](https://github.com/rust-mobile/android-view) was handed over to the Rust Mobile organisation.
+
+## Get Involved
+
+We welcome collaboration on any of our crates.
+This can include improving the documentation, implementing new features, improving our test coverage, or using them within your own code.
+
+We host an hour long office hours meeting each week where we discuss what's going on in our projects.
+See [#office hours in Zulip](https://xi.zulipchat.com/#narrow/channel/359642-office-hours) for details.
+We've also started a separate office hours time dedicated to the renderer collaboration, details also available at that link.
+
+- Daniel and Olivier's "office hours" appointments can still be booked by anyone for open-ended discussion of the ecosystem.
+  - [See Daniel's schedule here](https://calendar.google.com/calendar/u/0/appointments/schedules/AcZssZ32eQYJ9DtZ_wJaYNtT36YioETiloZDIdImFpBFRo5-XsqGzpikgkg47LPsiHhpiwiQ1orOwwW2).
+  - [See Olivier's schedule here](https://calendar.google.com/calendar/u/0/appointments/schedules/AcZssZ2t767ZRETD_TkRI_VxK2ZTG0VrO9OZ4l7HvTxefhtJcg85iK0ZN7zWNnAEZtH0Dn7C1GKxrmYM).
```

Comment:
I think I'll remove that calendar item soon, and I'd like to be removed from this section.

[#linebender > This month in Linebender @ 💬](https://xi.zulipchat.com/#narrow/channel/419691-linebender/topic/This.20month.20in.20Linebender/near/527698121)